front/tests/007-op-rollingstock-tab.spec.ts

```diff
@@ -7,22 +7,22 @@ import type {
   RollingStock,
   Scenario,
   Study,
-  Timetable,
+  TimetableResult,
 } from 'common/api/osrdEditoastApi';
 
 import scenarioData from './assets/operationStudies/scenario.json';
 import HomePage from './pages/home-page-model';
 import OperationalStudiesPage from './pages/operational-studies-page-model';
 import RollingStockSelectorPage from './pages/rollingstock-selector-page';
 import ScenarioPage from './pages/scenario-page-model';
-import { getProject, getStudy, getRollingStock, postApiRequest, getInfra } from './utils/index';
+import { getInfra, getProject, getRollingStock, getStudy, postApiRequest } from './utils/index';
 
 let smallInfra: Infra;
 let project: Project;
 let study: Study;
 let scenario: Scenario;
 let rollingStock: RollingStock;
-let timetable: Timetable;
+let timetable_result: TimetableResult;
```

Comment:
We don't use snake case in the front :) timetable or timetableResult is better
```suggestion
let timetable: TimetableResult;
```

Reply:
Ah sorry, I slipped